metricbeat/module/rabbitmq/queues/_meta/data.json

```diff
@@ -0,0 +1,19 @@
+{
+    "@timestamp":"2016-05-23T08:05:34.853Z",
+    "beat":{
+        "hostname":"beathost",
+        "name":"beathost"
+    },
+    "metricset":{
+        "host":"localhost",
+        "module":"rabbitmq",
+        "name":"queues",
+        "rtt":44269
+    },
+    "rabbitmq":{
+        "queues":{
+            "example": "queues"
+        }
```

Comment:
It looks like some fields are missing from this example `data.json`, can you generate a complete one?